tac/governing-documents/project-lifecycle.md

```diff
@@ -142,3 +142,126 @@ state independent of its prior history.
 
 [TAC]: https://www.lfdecentralizedtrust.org/staff?department=technical_oversight_committee
 [Proposal Template]: https://lf-decentralized-trust.github.io/project-proposals
+
+
+# Lifecycle Transition Guidelines and Recommendations
+
+The TAC must evaluate a project's lifecycle state during its [annual review](./project-annual-review.md) and make one of the following decisions:
+- Maintain its state
+- Upgrade (e.g., *Incubation* to *Graduated*)
+- Downgrade (e.g., *Graduated* to *Dormant*)
+
+(*Note*: as a rough hierarchy, the *Graduated* state represents the highest maturity, followed by *Incubated*, followed by *Dormant*, followed by *Archived*.)
+
+The following criteria MUST be considered for each project when making one of the aforementioned decisions:
+- Legal
+- Diversity
+- Releases
+- Testing and Q/A
+- Security
+- Structure
+- Maintenance
+- Production
+- Documentation
+
+## Description and Breakup of Criteria
+
+Below is a guide for evaluating each of these criteria, with break-ups into sub-criteria that can be directly evaluated using one of three methods:
+1. Checking their OpenSSF Scorecard scores
+2. Checking their LFX Insights Best Practice Scores (produced by CLOMonitor)
+3. Manual inspection of code repositories
+
+(*Note*: The term "close to" below implies that a high (or highest) score is not mandatory but highly desired. If a project falls short on some such criteria but is otherwise mature and well-functioning, the TAC may ignore those criteria while encouraging the project's maintainers to strive for a higher score.)
+
+### Legal Criteria
+
+- (OpenSSF Score): *License*: MUST be 10
+
+### Diversity Criteria
+
+- (OpenSSF Score): *Contributors*: SHOULD be close to 10
+- (Manual): the `MAINTAINERS.md` files in the project's repositories MUST collectively indicate the involvement of more than one organization
+
+- Evaluation:
+  * To be a *Graduated* project, the number of organizations from which maintainers are drawn must be at least 3, and the OpenSSF Score SHOULD be 10.
+  * To be an *Incubated* project, the number of organizations from which maintainers are drawn must be at least 2.
+
+### Releases
+
+- (OpenSSF Score): *Packaging*: SHOULD be 10
+- (Manual): the number and frequency of releases in the recent past (quarter/year), as identified by inspecting the project's repositories, must be reasonably high (at least for one of the repositories)
```

Comment:
We should define what we mean by `reasonably high` in this statement. Would 2 releases per year be a reasonable number?

Reply:
I wasn't sure what number to put without canvassing the TAC's opinion, hence the weasel words.

2 sounds very reasonable and a bare minimum, even for a mature and stable project that's not incorporating new features (there have to be bug fixes and dependency upgrades that necessitate new releases).

Anybody else have a strong opinion on this?